src/features/offer/components/OfferHeader.tsx

```diff
@@ -43,9 +44,14 @@ export const OfferHeader: React.FC<Props> = (props) => {
     onSuccess: () => {
       analytics.logHasAddedOfferToFavorites(params.from)
     },
-    onError: () => {
+    onError: (error) => {
       showErrorSnackBar({
-        message: _(t`L'offre n'a pas été ajoutée à tes favoris`),
+        message:
+          isApiError(error) && error.content.code === 'MAX_FAVORITES_REACHED'
+            ? _(
+                t`Trop de favoris enregistrés. Supprime des favoris pour pouvoir en ajouter de nouveaux`
```

Comment:
Petite faute sur "de nouveau" (j'admets que j'ai dû vérifier moi-même avant, donc pas facile à spotter :D) 
```suggestion
                t`Trop de favoris enregistrés. Supprime des favoris pour en ajouter de nouveau`
```

Reply:
C'est un copier coller, est-ce qu'on fait valider avant quand même ?

Reply:
I see. Je te laisse décider

Reply:
Lina est OK